### PR TITLE
Build as release

### PR DESCRIPTION
**Pull request by danger89**
By default it was building a debug build. I think this is not wanted when running in production.

Regards,
Melroy vd Berg

### CODE REVIEW

File: docs/building.rst [hunk 1]
```diff
@@ -72,21 +72,21 @@ Linux::
 
 	sudo apt install libboost-tools-dev libboost-dev libboost-system-dev
 	echo "using gcc ;" >>~/user-config.jam
-	b2 crypto=openssl cxxstd=14
+	b2 crypto=openssl cxxstd=14 variant=release
```
Comment (arvidn) [inline docs/building.rst:75]: `variant=` is not needed, you can just say `release`. I would prefer that.

**Review by arvidn — CHANGES_REQUESTED** [review]
looks good apart from the comment

File: docs/building.rst (hunk 1 above)
Comment (danger89) [reply docs/building.rst:75]: if you say so.

**Comment by danger89** [comment]
reworked

**Pull request merged**